### DISCUSSION TITLE
How to change the app productName in rust at runtime?

### DISCUSSION THREAD

**Discussion by lavamini** [tauri-apps/tauri#5964] [Q&A]
Hi, All
I am learning Tauri right now, it is so great!

In my app, I need to change the app productName in rust at runtime:
when using English language, the app display English name,
when using Other language, the app display Other language name

Thanks

**Comment by FabianLars** [comment]
That's not possible, but it's basically only used at build time (to set/get the binary name etc). What exactly are you trying to change?

**Reply by lavamini** [reply]
Thanks FabianLars, I am trying to Localize my macOS Bundle Display Name.

**Reply by bbqsrc** [reply]
Yes, this is also necessary for work I do. We need to be able to localise the app name since it shows up in the global menu bar.
